website/.prettierignore

```diff
@@ -0,0 +1,3 @@
+.eslintrc.json
```

Comment:
This didn't seem to have an effect on pre-commit.  I can later exclude these from pre-commit checks.

Reply:
Yeah, dont know why running prettier with pre-commit doesn't apply the ignore config.